pkgs/servers/nosql/mongodb/5.0.nix

```diff
@@ -6,22 +6,24 @@ let
   };
   variants = if stdenv.isLinux then
     {
-      version = "5.0.7";
-      sha256 = "sha256-1PeDBZJNqJXHH/cSh2e+WR0PfS/b7XuJEzkkbrRT/gc=";
+      version = "5.0.18";
+      sha256 = "sha256-tvQkDBwXYRZbIuST49JJ5T9zzYe/4BQ8ul1vUGlXHxI=";
+      patches = [ ./fix-build-with-boost-1.79-5_0-linux.patch ];
     }
-  else
```

Comment:
The elseif was good: it handle netbsd https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix#L37-40 here we get an empty attr set and `variants.patches` cause a missing attribute on non-Darwin/non-Linux platforms. If you tested on such platforms and it doesn't build consider throwing to avoid eval error, if the patches is not needed consider putting an empty list instead.